hoss/spatial.py

```diff
@@ -116,6 +122,22 @@ def get_spatial_index_ranges(
                     )
                 )
 
+        if (not geographic_dimensions) and (not projected_dimensions):
+            coordinate_variables = get_coordinate_variables(varinfo, required_variables)
+            if coordinate_variables:
+                for non_spatial_variable in non_spatial_variables:
+                    index_ranges.update(
+                        get_x_y_index_ranges_from_coordinates(
+                            non_spatial_variable,
+                            varinfo,
+                            dimensions_file,
+                            coordinate_variables,
```

Comment:
I think my other comments about multiple grids might be solved if the content of `coordinate_variables` was different. It looks like this is every coordinate variable for all required variables (from the declaration just above). If it was just the coordinate references for the single `non_spatial_variable`, then you shouldn't have to worry about multiple grids.

Reply:
updated - https://github.com/nasa/harmony-opendap-subsetter/pull/15/commits/802fe0ec04b853a7a7abb4718e2a05b0e8e6ed3c